### PR TITLE
DOC: Add documentation  for html-noplot

### PR DESCRIPTION
**Pull request by dipinknair**
Documentation can be generated without running examples. This PR updates required instructions.

### CODE REVIEW

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: doc/source/how-to/documenting.rst [hunk 1]
```diff
@@ -260,6 +260,34 @@ Because these examples are built using Sphinx-Gallery, you must follow its codin
...
+      @echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+
+Alternatively, you can modify ``sphinx_gallery_conf`` in ``conf.py`` as shown below
+
+.. code:: Python
+
+    sphinx_gallery_conf = {
+      ...
+      'plot_gallery': 'False',
+    }
```
Comment (RobPasMue) [inline doc/source/how-to/documenting.rst:289]: This only works if you are using sphinx-gallery. There are some libraries that do not use it. Please adapt accordingly to include the disclaimer.

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: doc/source/how-to/documenting.rst (hunk 1 above)
Comment (dipinknair) [reply doc/source/how-to/documenting.rst]: Updated the section accordingly.

File: doc/source/how-to/documenting.rst [hunk 2]
```diff
@@ -260,6 +260,35 @@ Because these examples are built using Sphinx-Gallery, you must follow its codin
...
+includes examples generated with the assistance of Ansys products.
+Building documentation locally is time consuming in these cases,
+particularly if you need to test changes only in the documentation excluding the examples.
+
+To generate documentation without examples, use command ``make html-noplot`` with
+appropriate changes in ``make.bat`` and ``Makefile``. Change needed for ``Makefile``
+is mentioned below. Refer `sphinx-gallery documentation 
+<https://sphinx-gallery.github.io/stable/configuration.html#building-without-executing-examples>`_.
+
+.. code:: shell
```
Comment (Revathyvenugopal162) [inline doc/source/how-to/documenting.rst:276]: making this as a tab with makefile and make.bat file would be helpful

**Review by RobPasMue — APPROVED** [review]

File: doc/source/how-to/documenting.rst (hunk 2 above)
Comment (RobPasMue) [reply doc/source/how-to/documenting.rst:276]: I completely agree

**Pull request merged**